src/routers/TabRouter.js

```diff
@@ -32,6 +32,8 @@ export default (
   const initialRouteIndex = order.indexOf(initialRouteName);
   const backBehavior = config.backBehavior || 'initialRoute';
   const shouldBackNavigateToInitialRoute = backBehavior === 'initialRoute';
+  const shouldBackNavigateToLastActiveTab = backBehavior === 'previousRoute';
+  const backstack = [];
```

Comment:
This stores the state in the router, which is not what we want!

Instead we should store the back stack as routes within the state that we return from `getStateForAction`, (which also comes in through inputState)

Please keep the routers stateless! Sometimes the router needs to get re-constructed (like if the app is refreshed), but we still don't want to loose the back stack!